renovate.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+    
+        ":pinVersions",
+        ":automergeMinor",
+        ":rebaseStalePrs",
+    
+        "group:linters"
+      ],
+      "labels": ["dependencies"],
+      "automergeStrategy": "fast-forward",
+      "bumpVersion": "patch"
```

Comment:
I personally like this a lot - whenever it updates a dependency, it also updates the main package version

Although I think this is mainly useful for web, where every commit to main also is a release. Might make sense to not include this for you, If you cant every prod release to be at max a single version bump (either major,minor,or patch) and not a combination of potentially many